Tests/.editorconfig

```diff
@@ -1,149 +1,149 @@
 [*.cs]
 
 # IDE0051: Private member is unused
-dotnet_diagnostic.IDE0051.severity = none
+dotnet_diagnostic.ide0051.severity = none
```

Comment:
Why this change?

Reply:
Rider did that. Apparently that's the convention it used... 🤷🏻

Reply:
Hmm... their [website](https://www.jetbrains.com/help/rider/Using_EditorConfig.html#what_is) uses uppercase like any other `.editorconfig` I've ever seen.

Reply:
I removed the entire file from the PR.